src/TransitionViz.scss

```
@@ -1,12 +1,15 @@
 [data-viz='transition'] {
-  --viz-transition-color: gray;
+  --viz-transition-color: #333;
   display: block;
-  border-radius: 1rem;
+  border-radius: 0.4rem;
```

```suggestion
  border-radius: 1rem;
```

I feel that we should have transitions visually distinct from the states in overall shape, with more rounded corners 🤔

---

I don't think it makes much of a visual difference, and I find the over-roundedness a bit ugly - if we do round them to 1rem we'll need to increase the padding.

---

Alright, let's increase the left/right padding 👍